pkg/cri/config/config.go

```diff
@@ -24,6 +24,9 @@ import (
 	"time"
 
 	"github.com/containerd/log"
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1"
+
+	"github.com/containerd/containerd/pkg/cri/annotations"
```

Comment:
We don't have goimport lint yet. But I think you should group `github.com/containerd/*` packages together.

Reply:
And it's weird to put the `cri-api` deps in the config. What about holding the helper `GetSandboxRuntime` in the `pkg/cri/utls`?

Reply:
> We don't have goimport lint yet. But I think you should group github.com/containerd/* packages together. 
It seems the rule of goimports organize imports in 3 groups, 1. packages in runtime, like "io", "fmt", 2. package of other projects, 3. packages in the same project. the "github.com/containerd/log" seems to be "package in other project", because it is an independent github project.

Reply:
> And it's weird to put the cri-api deps in the config. What about holding the helper GetSandboxRuntime in the pkg/cri/utls?

I tried but it will make a cyclic dependency if we define GetSandboxRuntime in github.com/containerd/containerd/pkg/cri/util